client/starwhale/core/model/view.py

```diff
@@ -113,6 +120,37 @@ def history(self, fullname: bool = False) -> t.List[t.Dict[str, t.Any]]:
             title="Model History List", history=self.model.history(), fullname=fullname
         )
 
+    @classmethod
+    def run_in_server(
+        cls,
+        project_uri: URI,
+        model_uri: str,
+        dataset_uris: t.List[str],
+        runtime_uri: str,
+        resource_pool: str,
+        run_handler: str | int,
+    ) -> t.Tuple[bool, str]:
+        ok, version_or_reason = CloudModel.run(
+            project_uri=project_uri,
+            model_uri=model_uri,
+            dataset_uris=dataset_uris,
+            runtime_uri=runtime_uri,
+            resource_pool=resource_pool,
+            run_handler=run_handler,
+        )
+        if ok:
+            console.print(":clap: success to create job")
+            console.print(
+                f":bird: visit web: {project_uri.full_uri}/evaluation/{version_or_reason}"
```

Comment:
The current eval job details is: http://{ip}/projects/{projectId}/evaluations/{jobId}

Reply:
fixed

Reply:
👏 select server instance
👏 success to create job
🐦 visit web: http://127.0.0.1:8082/projects/starwhale/evaluations/1
🐒 run command: swcli job info http://127.0.0.1:8082/project/starwhale/job/1